### PR TITLE
Feature - Show PreInstall Notes on Install Modal

### PR DESCRIPTION
**Pull request by daiyiwhale**
Collapse-o

Expand-y

### CODE REVIEW

File: src/js/components/modals/InstallPackageModal.js [hunk 1]
```diff
@@ -293,14 +305,60 @@ class InstallPackageModal
     );
   }
 
+  getPreinstallNotes(notes, truncated) {
+    if (truncated && notes) {
```
Comment (kennyt) [inline src/js/components/modals/InstallPackageModal.js]: wanna return `null` if `!notes`? then we won't have this basically empty `p` tag

**Comment by kennyt** [comment]
after my last comment, I think this looks good!

File: src/js/components/modals/InstallPackageModal.js (hunk 1 above)
Comment (daiyiwhale) [inline src/js/components/modals/InstallPackageModal.js]: done!

**Comment by daiyiwhale** [comment]
👍

**Comment by kennyt** [comment]
@mlunoe what do you think about this PR? I remember you had some comments before

**Comment by mlunoe** [comment]
Looks good and works great! :shipit:

**Pull request merged**